conf/app.routes

```diff
@@ -9,4 +9,6 @@ PUT    /overseas-subscription                 @uk.gov.hmrc.agentsubscription.con
 
 GET    /corporation-tax-utr/:utr/crn/:crn     @uk.gov.hmrc.agentsubscription.controllers.CTReferenceController.matchCorporationTaxUtr(utr:uk.gov.hmrc.agentmtdidentifiers.model.Utr, crn:uk.gov.hmrc.agentsubscription.model.Crn)
 
-GET    /vat-known-facts/vrn/:vrn/dateOfRegistration/:dateOfReg     @uk.gov.hmrc.agentsubscription.controllers.VatKnownfactsController.matchVatKnownfacts(vrn:uk.gov.hmrc.domain.Vrn, dateOfReg:String)
\ No newline at end of file
+GET    /vat-known-facts/vrn/:vrn/dateOfRegistration/:dateOfReg     @uk.gov.hmrc.agentsubscription.controllers.VatKnownfactsController.matchVatKnownfacts(vrn:uk.gov.hmrc.domain.Vrn, dateOfReg:String)
+
+POST    /citizen-details                     @uk.gov.hmrc.agentsubscription.controllers.CitizenDetailsController.checkCitizenDetails()
```

Comment:
since its  a api call, its easy if you read the params as query params as above , it makes our life easy as you dont need to parse json using read/write format

Reply:
also you can make a GET call rather than POST as we are not creating any record in the db

Reply:
Since we are sending a Nino and Dob combination we wouldn't want this data appearing in Kibana logs. Talked with JR and he agreed with this.